packages/cron/src/index.ts

```diff
@@ -60,15 +60,20 @@ export class AgendaCronJobs {
 		this.reservedJobs = [];
 	}
 
-	public async add(name: string, schedule: string, callback: () => any | Promise<any>): Promise<void> {
+	public async add(name: string, schedule: string | Date, callback: () => any | Promise<any>): Promise<void> {
 		if (!this.scheduler) {
 			return this.reserve(name, schedule, callback);
 		}
 
 		this.scheduler.define(name, async () => {
 			await runCronJobFunctionAndPersistResult(async () => callback(), name);
 		});
-		await this.scheduler.every(schedule, name, {}, {});
+
+		if (typeof schedule === 'string') {
+			await this.scheduler.every(schedule, name, {}, {});
+		} else {
+			await this.scheduler.schedule(schedule, name, {});
```

Comment:
More of a semantic thing, but if you plan to add single use jobs, why not using your own agenda 'instance' and controlling the jobs there? 👀